### ISSUE TITLE
Seed unbeatable when DoT closed

### ISSUE THREAD

**Issue by matthevvv** [TestRunnerSRL/OoT-Randomizer#1351]
Tried the following seed: https://ootrandomizer.com/seed/get?id=836362

With the settings set to DoT closed, Spirit Temple is treated as a Child dungeon and the Goron Ruby is locked behind beating Twinrova, which cannot be done in logic since you can't defeat Twinrova as child and you cannot become adult in glitchless without the ruby.

Zora's Fountain is also closed and Ruto's letter is in Ganon's castle so it is also impossible to get the sapphire.

Basically, the logic assumes you can become adult at anytime, though you can't without glitching.

**Comment by ETR-BTF** [comment]
You only need _an_ Ocarina and Song of Time to open the door. :)

**Comment by Star1468** [comment]
This is the same as the vanilla game- the stones only unlock the Song of Time, not the door itself. 
This wiki page lists that and other useful information to know:
https://wiki.ootrandomizer.com/index.php?title=Logic

**Comment by matthevvv** [comment]
Huh, i tried that but it didn't work, may not have been in the right place?

**Comment by ETR-BTF** [comment]
> Huh, i tried that but it didn't work, may not have been in the right place?

Stand on the red carpet in front of the pedestal. If I remember correctly a note staff should pop up.
